x/chainlink/keeper/querier_test.go

```diff
@@ -0,0 +1,267 @@
+// Copyright 2021 ChainSafe Systems
+// SPDX-License-Identifier: MIT
+
+package keeper
+
+import (
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/ChainSafe/chainlink-cosmos/x/chainlink/types"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/testutil/testdata"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func TestQuerier_GetRoundFeedData(t *testing.T) {
+	keeper, ctx := setupKeeper(t)
+	amino := codec.NewLegacyAmino()
+	querier := NewQuerier(*keeper, amino)
+	roundStore := ctx.KVStore(keeper.roundStoreKey)
+
+	testCases := []struct {
+		feedId    string
+		roundId   uint64
+		feedData  []byte
+		submitter []byte
```

Comment:
submitter should be of type `sdk.AccAddress`
https://github.com/ChainSafe/chainlink-cosmos/blob/master/x/chainlink/types/tx.pb.go#L723